specification/content.md

```diff
@@ -43,13 +43,15 @@ content:
       ...
 ```
 
-## The Schema Object
+## The Schema Field
 
-The [Schema Object](https://spec.openapis.org/oas/v3.1.0#schema-object) defines a data type which can be a primitive (integer, string, ...), an array or an object depending on its `type` field.
+The schema field holds a [Schema Object](https://spec.openapis.org/oas/v3.1.0#schema-object).
 
-`type` is a string and its possible values are: `number`, `string`, `boolean`, `array` and `object`. Depending on the selected type a number of other fields are available to further specify the data format.
+Schema objects describe the structure of data, and may be nested to describe complex arrays and objects. They are most often used to describe JSON data.
```

Comment:
I think it's good to remove the "Depending on the selected type..." language as that's not really how JSON Schema works (you can use all fields at all times, it's just that they don't all apply to every data type, and some combinations are nonsensical- but they are still technically valid).

I'd just leave it as "Schema objects describe the structure of data."  The exact relationship between JSON Schema and non-JSON data varies a bit among JSON Schema and OpenAPI versions.